CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.1.0
+ * add interface for `interval` and `interval-after` interfaces to the
```

Comment:
I think you only need one "interface" in this sentence.

Reply:
department of redundancy department! :)